pyproject.toml

```
@@ -14,7 +14,7 @@ requires-python = ">=3.8"
 license = { file = "LICENSE" }
 dependencies = [
     "numpy",
-    "torch>=2.1,<2.4",
+    "torch>=2.4",
```

I don't think we should disallow earlier torch versions.
```suggestion
     "torch>=2.1,<2.5",
```